spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/DefaultSource.scala

```diff
@@ -23,10 +23,10 @@ import DefaultSource._
  *      CREATE TEMPORARY TABLE tmpTable
  *      USING org.apache.spark.sql.cassandra
  *      OPTIONS (
- *       c_table "table",
+ *       table "table",
  *       keyspace "keyspace",
  *       cluster "test_cluster",
- *       push_down "true",
+ *       pushdown "true",
  *       spark_cassandra_input_page_row_size "10",
  *       spark_cassandra_output_consistency_level "ONE",
```

Comment:
I agree. Keyspace/table/cluster/pushdown should work now, others will not. But there were no tests for these settings before, so maybe let's address it with a separate ticket we'd do after we hear back from DataBricks.